commit d62483f433c9a7631185aa2b93f9b8b6297d905c (refs/changes/92/2392/1)
Author: Angela Fong <angela.fong@joyent.com>
Date:   2017-08-14T16:21:59-07:00 (2 years, 2 months ago)
    
    MANTA-3373 Docs: Need to review/fix bad links in API docs

diff --git a/docs/user-guide/jobs-reference.md b/docs/user-guide/jobs-reference.md
index 2372326..e11450d 100644
--- a/docs/user-guide/jobs-reference.md
+++ b/docs/user-guide/jobs-reference.md
@@ -327,7 +327,7 @@ reducers in the next phase.  Also see documentation for "msplit".
 ## Compute instance images ("image" property)
 
 Compute instances are Joyent instances based on the manta-compute image. This
-image is essentially a [base image](https://wiki.joyent.com/wiki/display/jpc2/SmartMachine+Base)
+image is essentially a [base image](https://docs.joyent.com/public-cloud/instances/infrastructure/images/smartos/base)
 with nearly all of the available packages preinstalled.
 
 By default, tasks run in compute instances with the most recently released image
